user-interface/src/main/java/life/qbic/datamanager/views/general/footer/DataPrivacyAgreement.java

```diff
@@ -33,7 +34,7 @@ public class DataPrivacyAgreement extends Main {
   public DataPrivacyAgreement() {
     String dataPrivacyAgreementHtmlContent = getDataPrivacyAgreementHtmlContent();
     // Replace href="#" with href="currentPath#id"
-    String dataPrivacyAgreementWithAnchors = dataPrivacyAgreementHtmlContent.replaceAll(
+    String dataPrivacyAgreementWithAnchors = dataPrivacyAgreementHtmlContent.replace(
```

Comment:
because replaceAll uses a pattern matcher and it is useless, if no regex is passed. Unnecessary performance drawback.

https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#replaceAll(java.lang.String,%20java.lang.String)